bluemira/geometry/base.py

```diff
@@ -94,9 +94,9 @@ def __init__(
     ):
         super().__init__()
         self._boundary_classes = boundary_classes
-        self.boundary = boundary
-        self.label = label
         self.__orientation = _Orientation("Forward")
```

Comment:
`_orientation` and `__orientation` both exist. I don't think this is intended

Reply:
I agree this should always go through the setter of `_orientation`